app/views/maintenance_tasks/runs/_run.html.erb

```diff
@@ -8,4 +8,7 @@
   <% else %>
     <%= render 'maintenance_tasks/runs/info', run: run, with_status: true %>
   <% end %>
+  <% if defined?(task) && task.csv_task? %>
```

Comment:
This is not so nice, but is hard to achieve otherwise given that this partial is used to show the list of previous runs, but also to power the Runs index page (in which Runs are not scoped to Tasks). This will hide the download links from the Runs index page (based on https://github.com/Shopify/maintenance_tasks/pull/288 I'm not sure we're intending to keep this anymore?). If we wanted to show this link on the index page as well, I think we could modify it to be:

```ruby
 <% if defined?(task) && task.csv_task? || TaskData.find(run.task_name).csv_task? %>
```

But not sure I like that...